control-plane/config/sink/100-kafka-sink.yaml

```diff
@@ -58,6 +58,21 @@ spec:
                 bootstrapServers:
                   description: 'A comma separated list of host/port pairs to use for establishing the initial connection to the Kafka cluster.'
                   type: string
+                encoding:
+                  description: 'CloudEvent content mode of Kafka messages sent to the topic.
+                                Possible values:
+                                  - structured
+                                  - binary
+
+                                - default: structured.
+
+                                - https://github.com/cloudevents/spec/blob/master/spec.md#message
```

Comment:
Better if you point to v1 spec links https://github.com/cloudevents/spec/blob/v1.0/spec.md

Reply:
Done: https://github.com/knative-sandbox/eventing-kafka-broker/pull/134/commits/8ccbd970c378ebd8b938c3d413349b98e224eb35